oaknational/Oak-Web-Application
# feat(video): create standalone video page

## Description

Music year: 2012

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2456--oak-web-application.netlify.thenational.academy/videos/mgQ9kDg302EyPWi5aKh00hTgLy9xwG2ud8U82R9uVw68M/player
2. Click on the play button
3. You should see videos working for correct playback ids

### Job Story

**When** a commercial partner uses an oak video,
**I want the to use a none branded mux video player page**
**So that use can** use oak video assets on different applications.

### Implementation details

serve from /api/videos/[videoId]/player

Use the existing mux player and just feed in the muxId

### Acceptance criteria

- [x]  plays video with correct id
- [x]  provides suitable information for an incorrect id

## Checklist
